lib/bolt/outputter/human.rb

```diff
@@ -344,7 +344,7 @@ def print_targets(target_list, inventoryfile)
         end
 
         @stream.puts "INVENTORY FILE:"
-        if inventoryfile.exist?
+        if File.exist?(inventoryfile.to_s)
```

Comment:
`to_s` shouldn't be necessary here.

Reply:
I think I now understand your inclination toward Pathname. It would indeed be a pain if we had to call `to_s` everywhere (the same pain as having to call `Pathname.new` everywhere).

`Pathname` can actually be used _interchangeably_ with strings for the `File` methods, which is why I prefer it for the cases where we aren't using anything that's `Pathname`-specific (`parent`, `relative_path_from`, `+`).

Reply:
Ah, ok - yeah, I do hate that when I print a pathname it doesn't know that I just want the path, not `<Pathname:/path>`. Assumed it needed the to_s, since it doesn't make we're fine using both mixed for now. I was thinking we could just ensure all paths in the Config class are pathnames, similar to in Project, but it's definitely unnecessary there since we aren't building paths. It would just let us make more assumptions like the one here.